packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -0,0 +1,364 @@
+        /**
+         * Actions for "Accept only required cookies" button
+         */
+        nonAcceptActions () {
+            this.setCookieBannerCookie('necessary');
+            this.setLegacyCookieBannerCookie();
+            this.dataLayerPush('necessary');
+            this.resendEvents();
+            this.removeUnnecessaryCookies();
+            this.hideBanner = true;
+        },
+        /**
+         * Set focus to the cookie consent banner title for accessibility
+         */
+        focusOnTitle () {
+            this.$refs.cookieBannerHeadingFocus.focus();
+        },
+        /**
+         * Check if the cookie banner has been shown to this user
+         */
+        checkLegacyBannerFlag () {
+            this.legacyBanner = this.config.displayLegacy;
+        },
+        /**
+         * Check if the cookie banner has been shown to this user
+         */
+        checkCookieBannerCookie () {
+            if (this.legacyBanner) {
+                this.hideBanner = this.$cookies.get('je-banner_cookie') === 2;
+                this.setLegacyCookieBannerCookie();
+            } else {
+                const cookieConsent = this.$cookies.get('je-cookieConsent');
+                this.hideBanner = cookieConsent === 'full' || cookieConsent === 'necessary';
+            }
+        },
+        /**
+         * Set the cookie for the user's choice
+         * @param {String} cookieValue
+         */
+        setCookieBannerCookie (cookieValue) {
+            this.$cookies.set('je-cookieConsent', cookieValue, {
+                path: '/',
+                maxAge: this.cookieExpiry
+            });
+        },
+        /**
+         * Set the legacy cookie banner cookie
+         */
+        setLegacyCookieBannerCookie () {
+            this.$cookies.set('je-banner_cookie', '2', {
+                path: '/',
+                maxAge: this.cookieExpiry
+            });
+        },
+        /**
+         * Push tracking events
+         * @param {String} consentLevel
+         */
+        dataLayerPush (consentLevel) {
+            const dataLayer = window.dataLayer || [];
+            dataLayer.push({ event: 'trackConsent', userData: { consent: consentLevel } });
+        },
+        /**
+         * Check for excluded cookies/storage
+         * @param {String} cookieName
+         * @returns {Bool}
+         */
+        isNotExcluded (cookieName) {
+            return this.config.cookieExclusionList.every(arrVal => cookieName.lastIndexOf(arrVal, arrVal.length - 1) === -1);
+        },
+        /**
+         * Remove unnecessary cookies
+         */
+        removeUnnecessaryCookies () {
+            const cookies = Object.keys(this.$cookies.getAll());
+            for (let i = 0; i < cookies.length; i++) {
+                if (this.isNotExcluded(cookies[i])) this.$cookies.remove(cookies[i]);
+            }
+        },
+        /**
+         * Check for iOS browser
+         * @returns {Bool}
+         */
+        checkIosSafari () {
```

Comment:
Same as above. This to me should be named `isIosBrowser` or similar.

Reply:
Done